content/updates/2020-11/1-Stop-your-imports-and-exports.md

```diff
@@ -0,0 +1,24 @@
+---
+pim_announcement_alt_img: "The new Stop button in the process tracker"
+pim_announcement_audience:
+- EE
+- CE
+---
+
+# Stop your imports and exports
+::: meta-data type="New" features="Productivity" available="mid-November" link-to-doc="../articles/TODO.html"
+:::
+
+A brand new `Stop` button is now displayed in your process tracker. It enables you to stop the Excel and CSV imports/exports you launched. Super helpful whenever you launch a job by mistake and don't want to wait for it to execute until the end. 😉
```

Comment:
```suggestion
A`Stop` button is now displayed close to jobs in your process tracker. It enables you to stop jobs: imports, exports but also ongoing mass actions or rule executions. This new feature is controlled with permissions, so if you want to be able to stop jobs, ask your admin. Super helpful whenever you or someone from your team launches a job and needs to stop it.
```
@felix-couprie is this correct ⬆️? thank you!

Reply:
```suggestion
A brand new `Stop` button is now displayed in your process tracker. It enables you to stop jobs: imports, exports but also ongoing mass actions or rule executions. Super helpful whenever you launch a job by mistake and don't want to wait for it to execute until the end. 😉
```

Reply:
```suggestion
A brand new `Stop` button is now displayed in your process tracker. It enables you to stop jobs: imports, exports but also ongoing mass actions or rule executions. Super helpful whenever you launch a job by mistake (we're all humans after all) and don't want to wait for it to execute until the end. 😉
```